datasets/fhad/make_data.py

```diff
@@ -10,8 +10,16 @@
 # Set Path here
 # EXAMPLE:
 #   ROOT = '/path/to/FHAD/hand_pose_action'
+import os
+import sys
 
-ROOT = None
+ROOT = "/mnt/batch/tasks/shared/LS_root/mounts/clusters/wack/code/Users/aengus.crowley/FHAD/First-Person Hand Action Benchmark (F-PHAB)"
+<<<<<<< HEAD
```

Comment:
This code won't run.   It looks like something got merged mid-diff.   Everything between `<<<<<<< HEAD` and `>>>>>>> 6bf9b42 (stash commit I think)` needs to be trimmed up.